draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -1079,6 +1101,40 @@ following Payload-specific RTP Feedback (PSFB) feedback.
   we have QUIC timestamps, we don't need to work around RTP timestamps offsets
   because we can use the QUIC timestamps to calculate network jitter.
 
+# Error Handling {#error-handling}
+
+The following error codes are defined for use when abruptly terminating streams,
+aborting reading of streams, or immediately closing RoQ connections.
+
+ROQ\_NO\_ERROR (0x????):
+: No error. This is used when the connection or stream needs to be closed, but
+there is no error to signal.
+
+ROQ\_INTERNAL\_ERROR (0x????):
+: An internal error has occured in the RoQ stack.
+
+ROQ\_PACKET\_ERROR (0x????):
+: Invalid payload format, e.g., length does not match packet, invalid flow id
+encoding, non-RTP on RTP-flow ID, etc.
+
+ROQ\_STREAM\_CREATION\_ERROR (0x????):
+: The endpoint detected that its peer created a stream that it will not accept.
```

Comment:
The use of this error code in the document, in 

> To send RTP/RTCP packets over QUIC streams, a sender MUST open a
> new unidirectional QUIC stream. Streams are unidirectional because there is no
> synchronous relationship between sent and received RTP/RTCP packets. A peer that
> receives a bidirectional stream with a flow identifier that is associated with
> an RTP or RTCP stream, SHOULD stop reading from the stream and send a
> STOP\_SENDING frame with the application protocol error code set to
> ROQ\_STREAM\_CREATION\_ERROR.

describes this situation as a fundamental protocol violation. I read 

> ROQ\_STREAM\_CREATION\_ERROR (0x????):
> : The endpoint detected that its peer created a stream that **it will not accept.**

as more of a decision by the endpoint, rather than what the endpoint does to conform to the protocol specification. 

Maybe something like 

> ROQ\_STREAM\_CREATION\_ERROR (0x????):
> : The endpoint detected that its peer created a stream that **violates the ROQ protocol.**